docs/pages/index.mdx

```diff
@@ -57,9 +63,223 @@ endpoints, infrastructure assets, and AI agents.
+        }
+      ]
+    },
+    {
+      id: 'machine-workload-identity',
+      title: 'Machine and Workload Identity',
+      href: './machine-workload-identity/',
+      description: 'Replace long-lived secrets with identity-based authentication and authorization',
+      iconColor: '#EEEAFA',
+      iconComponent: MWISvg,
+      features: [
+        {
+          title: 'Intro to Machine & Workload ID',
+          description: 'Replace long-lived secrets with identity-based auth',
+          href: './machine-workload-identity/'
+        },
+        {
+          title: 'Deploy CI/CD Pipelines',
+          description: 'Replace long-lived secrets in CI/CD pipelines',
+          href: './machine-workload-identity/machine-id/deployment/#cicd'
+        },
+        {
+          title: 'Secure Infrastructure as Code',
+          description: 'Manage IaC workflows in Terraform and Pulumi',
+          href: './admin-guides/infrastructure-as-code/terraform-provider/'
+        },
+        {
+          title: 'Hybrid & Multi-Cloud Authentication',
+          description: 'Universal identities across cloud platforms',
+          href: './machine-workload-identity/machine-id/deployment/'
+        },
+        {
+          title: 'Workload to Workload Authentication',
+          description: 'Service-to-service auth with mTLS',
+          href: './machine-workload-identity/workload-identity/spiffe/'
+        },
+        {
+          title: 'Identity Management for AI Agents',
+          description: 'RBAC for autonomous agents and processes',
+          href: './machine-workload-identity/workload-identity/introduction/'
+        }
+      ]
+    },
+    {
+      id: 'identity-governance',
+      title: 'Identity Governance',
+      href: './identity-governance/',
+      description: 'Manage identities by enforcing principles of least privilege and zero trust',
+      iconColor: '#EEEAFA',
+      iconComponent: IGSvg,
+      features: [
+        {
+          title: 'Request Temporary Elevated Access',
+          description: 'Eliminate standing privileges w/ just-in-time access',
+          href: './identity-governance/access-requests/'
+        },
+        {
+          title: 'Manage Standing Access for Teams',
+          description: 'Sync IdP groups to roles w/ automated reviews',
+          href: './identity-governance/okta/app-and-group-sync/'
+        },
+        {
+          title: 'Require Managed Devices for Access',
+          description: 'Guarantee access only from trusted devices',
+          href: './identity-governance/device-trust/'
+        },
+        {
+          title: 'Instantly Lock Identities & Sessions',
+          description: 'Lock compromised users and resources',
+          href: './identity-governance/locking/'
+        },
+        {
+          title: 'Integrate w/your Identity Provider(s)',
+          description: 'Okta, Entra ID, and Sailpoint w/SCIM group sync',
+          href: './identity-governance/okta/'
```

Comment:
The description mentions a variety of IdPs, but the link goes to an Okta page. Is that intentional?

If I saw Entra ID here and clicked the link I think I'd be surprised to land on a page about Okta.